src/test/java/uk/gov/hmcts/ccd/domain/service/aggregated/GetEventTriggerOperationTest.java

```diff
@@ -1,149 +0,0 @@
-package uk.gov.hmcts.ccd.domain.service.aggregated;
```

Comment:
This is duplicating a class that already exists and includes these tests: `DefaultGetEventTriggerOperationTest`.